### ISSUE TITLE
Query state of GitHub issues

### ISSUE THREAD

**Issue by wilzbach** [dlang/dlang-bot#259]
How to?

```graphql
{
  repository(name: "dub", owner: "dlang") {
    issue10: issue(number: 10) {state},
    issue12: issue(number: 12) {state},
  }
}
```

(-> https://developer.github.com/v4/explorer/)

However, the bot won't be able to know whether an issue reference is from GitHub or Bugzilla, so we could only enable this for repository with GitHub issues only (like `dub`).

**Comment by CyberShadow** [comment]
I don't understand what this issue is about :)

**Comment by Geod24** [comment]
I don't know if it's relevant anymore. Currently if you do `Fix #12345 - Title`, Github will automatically close things for you if there's an issue with the same ID. However, that creates an issue in `dub`, as the bot does not recognize that it should not take action there. I believe that might be what @wilzbach is talking about.

Example:
This PR: https://github.com/dlang/dub/pull/2103 contained "Fix 2046"
Which led to this: https://issues.dlang.org/show_bug.cgi?id=2046#c3
